Source/Core/DolphinWX/VideoConfigDiag.h

```diff
@@ -193,6 +193,8 @@ class VideoConfigDiag : public wxDialog
 		// Anti-aliasing
 		choice_aamode->Enable(vconfig.backend_info.AAModes.size() > 1);
 		text_aamode->Enable(vconfig.backend_info.AAModes.size() > 1);
+		if (vconfig.backend_info.bSupportsSSAA)
+			ssaa_checkbox->Enable(vconfig.iMultisampleMode > 0);
```

Comment:
Adding ssaa_checkbox = new wxCheckBox; if this conditional passes seems to resolve the crashing as well, and doesn't create a checkbox in Direct3D